server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -120,6 +125,7 @@ private void removeCompactColumnsFromRootTabletMetadata(ServerContext context) {
           Preconditions.checkState(key.getColumnVisibilityData().length() == 0,
               "Expected empty visibility, saw %s ", key.getColumnVisibilityData());
           Mutation m = new Mutation(row);
+          // TODO will metadata contraint fail when this is written?
```

Comment:
@EdColeman I going off of memory from doing code reviews and confused #3876 with #3963.  @dlmarion do you know if this write that deletes the compact col will fail because the metadata constraint does not know about it? Trying to determine if I need to open an issue for this.